docs/source/modules/convection_simple_betts_miller.rst

```diff
@@ -0,0 +1,95 @@
+
+
+Namelist options
+----------------
+
+There are five optional namelist variables. 
+
+The first two are directly relevant to how the tendency equations and precipitation are calculated
+
+:tau_bm: The relaxation time (:math:`\tau`) with the default value of :math:`7200` :math:`s`.
+:rhbm: The critical Relative Humidity (RH) used to calculate the moisture reference profile (:math:`RH_{SBM}`) with the default value of :math:`0.8`.
+
+The final three are used to construct the look up table for calculating the lifting condensation level (LCL).
+
+:Tmin: The minimum look up table value with the default of :math:`173` :math:`K`.
+:Tmax: The maximum look up table value with the default of :math:`335` :math:`K`.
+:val_inc: The increment used to linearly interpolate within the look up table with the default of :math:`0.01`.
+
+
+Diagnostics
+-----------
+The diagnostics are not sent out within the call to convection but are sent out from ``idealized_moist_phys.F90`` with the module name ``atmosphere``. The diagnostics relevant to the simple Betts-Miller scheme are in the table below.
+
++-------------------+----------------------------+------------------------------------+
+| Name              | Description                | Units                              |
++===================+============================+====================================+
+| dt_qg_convection  | Moisture tendency          |:math:`kg~kg^{-1}~s^{-1}`           |
++-------------------+----------------------------+------------------------------------+
+| dt_qg_convection  | Temperature tendency       |:math:`K~s^{-1}`                    |
++-------------------+----------------------------+------------------------------------+
+| convection_rain   | Convective precipitation   |:math:`kg~m^{-2}~s^{-1}`            |
++-------------------+----------------------------+------------------------------------+
+| cape              | Convective available       |:math:`J~kg^{-1}`                   |
+|                   | potential energy           |                                    |
++-------------------+----------------------------+------------------------------------+
+| cin               | Convective inhibition      |:math:`J~kg^{-1}`                   |
++-------------------+----------------------------+------------------------------------+
+
+
+Relevant modules and subroutines
+--------------------------------
+
+The convection scheme is located in: ``src/atmos_param/qe_moist_convection/qe_moist_convection.F90``. The files name reflects this style of a quasi-equilibirum (qe) convection scheme (for more information on convective quasi-equilibirum see [YanoPlant2016]_).
+
+The simple Betts-Miller scheme is initialised and called by: ``/src/atmos_spectral/driver/solo/idealized_moist_phys.F90``.
+
+Relevant routines which are called by the convection scheme are:
+``src/shared/sat_vapor_pres/sat_vapor_pres.F90``.
+
+
+
+References
+----------
+
+[Frierson2007]_
```

Comment:
Maybe put comma's after each reference or each on a new line? They show up in a string at the moment.